cmd/app/config.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	"parabellum.crawler/internal/crawler"
+)
+
+const pathToEnvFile = ".env"
+
+const (
+	TypeString = iota
+	TypeInt
+	TypeTimeSecond
+)
+
+var TestsFilters = map[string][crawler.NumOfBodyParams]bool{
+	"SQLI-check": fillTestFilter("SQLi", true, false),
+	"BA-check":   fillTestFilter("Broken auth", true, false),
+	"XSS-check":  fillTestFilter("XSS", true, true),
+	"LFI-check":  fillTestFilter("LFI", false, true),
+}
+
+func fillTestFilter(testName string, filters ...bool) [crawler.NumOfBodyParams]bool {
+	if len(filters) != crawler.NumOfBodyParams {
+		log.Panicln("Wrong filters number for test", testName)
+	}
+
+	var result [crawler.NumOfBodyParams]bool
+	result[crawler.HasFormTag] = filters[crawler.HasFormTag]
```

Comment:
You will have panic here

Reply:
Why should I if it's an array of a fixed size?
Not receiving it now.

Reply:
Should use make for create map
If define map using var and try to set up something you receive the panic, for example: https://go.dev/play/p/ar6G6wvchfu

Reply:
Ahh, yes you are right 
Sorry, my bad 
I did not see classic array for a long time)